### PR TITLE
Update next-step.md

### PR DESCRIPTION
**Pull request by ArshadAriff**
Updated next-step.md

### CODE REVIEW

**Comment by ArshadAriff** [comment]
@jcubic sir!

File: docs/docs/scheme-intro/next-step.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ A classic, one of the authors of the book is the author of Scheme language.
 ![SICP Cover](./img/SICP-cover.jpg)<br/>
 <small><a href="https://commons.wikimedia.org/wiki/File:SICP_cover.jpg">Harold Abelson and Gerald Jay Sussman with Julie Sussman — MIT Press</a>, <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>, Wikimedia Commons</small>
 
-I also recommend video lectures. There are two versions. I recommend original by Abelson and Sussman from 1986.
+I also recommend video lectures, with two versions available. The original by Abelson and Sussman from 1986 is highly recommended.
```
Comment (jcubic) [inline docs/docs/scheme-intro/next-step.md:25]: I don't like the phrase: "is highly recommended" here.

File: docs/docs/scheme-intro/next-step.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ There are also more recent lectures from 2010 at [Berkeley](https://www.berkeley
 
 ## Lisp Macros
 
-If you want to learn more about lisp macros, there are two great books:
+For more on Lisp macros, these two books are highly recommended:
```
Comment (jcubic) [inline docs/docs/scheme-intro/next-step.md:41]: Same, I don't like the "are highly recommended" and changing the sentence.

This doesn't look like a grammar fix.

**Review by jcubic — CHANGES_REQUESTED** [review]

**Comment by ArshadAriff** [comment]
Done Sir!

**Comment by ArshadAriff** [comment]
@jcubic anything else sir!

**Comment by ArshadAriff** [comment]
@jcubic sir i have only 10 mins left for the day to end sir!

**Comment by ArshadAriff** [comment]
@jcubic sir Hope to work with you in future!🙂

**Comment by ArshadAriff** [comment]
@jcubic sir i think hacktoberfest acceptable label is necessary for every PR submitted can you add it to my 4 PRs sir!

**Pull request merged**